Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m

```diff
@@ -160,16 +163,22 @@ - (void)updateTextDisplay {
         return;
     }
 
-    NSDictionary<NSAttributedStringKey, id> *newStringAttributes = [BPKFont attributesForFontStyle:self.fontStyle
-                                                                            withCustomAttributes:self.customFontAttributes
-                                                                                     fontMapping:self.fontMapping];
+    NSDictionary<NSAttributedStringKey, id> *newStringAttributes =
+        [BPKFont attributesForFontStyle:self.fontStyle
+                   withCustomAttributes:self.customFontAttributes
+                            fontMapping:self.fontMapping];
 
-    NSAttributedString *newString = [[NSAttributedString alloc] initWithString:self.text attributes:newStringAttributes];
+    NSAttributedString *newString = [[NSAttributedString alloc] initWithString:self.text
+                                                                    attributes:newStringAttributes];
     self.contentView.text = newString;
 
     // Re-apply the links
     for (BPKTappableLinkDefinition *linkDefinition in _persistedLinks) {
-        [self.contentView addLinkToURL:linkDefinition.url withRange:linkDefinition.range];
+        if (linkDefinition.hasURLDefinition) {
+            [self.contentView addLinkToURL:linkDefinition.url withRange:linkDefinition.range];
+        } else if (linkDefinition.hasTransitInformationDefinition) {
+            [self.contentView addLinkToTransitInformation:linkDefinition.components withRange:linkDefinition.range];
+        }
```

Comment:
Might be good to have an assert here in `else`

Reply:
done